include/dlr_common.h

```diff
@@ -108,6 +112,27 @@ class DLRModel {
   virtual const char* GetBackend() const = 0;
   virtual void SetNumThreads(int threads) = 0;
   virtual void UseCPUAffinity(bool use) = 0;
+
+  virtual bool HasMetadata() const {
+    return false;
+  };
+
+  virtual void GetOutputName(const int index, char*& name) const {
+    LOG(FATAL) << "GetOutputName is not supported yet!";
+  };
+
+  virtual int GetOutputIndex(const char* name) const {
+    LOG(FATAL) << "GetOutputIndex is not supported yet!";
+    return -1;
+  };
+
+  virtual void GetOuputByName(const char* name, float* out){
```

Comment:
missing space btw `){`. Better to use `clang-format -i --style=Google <file>`

Reply:
👍